### DISCUSSION TITLE
stubbing print window

### DISCUSSION THREAD

**Discussion by sagarrajput** [cypress-io/cypress#18323] [Questions and Help]
Hello
I trying to stub print preview but getting, could you please help, Ultimately I want to close print preview or somehow avoid from opening,
`expected print to have been called exactly once, but it was never called`
My code looks like
cy.window().then(win => {
cy.stub(win, 'print').as('print');
});
cy.get('my locator').click(); // this click event opens print widdow(Only after closing print preview window my API call happens).
cy.get('@print').should('have.been.calledOnce');
tagging: @brian-mann @jennifer-shehane @bahmutov

**Comment by navdeepdevrepublic** [comment]
@brian-mann @jennifer-shehane @bahmutov Can anyone help us with this.

@sagarrajput Do you have any workaround for this?

**Reply by sagarrajput** [reply]
@navdeepdevrepublic Unfortunately, No.
